dashboards/nvidia-gpu/README.md

```diff
@@ -15,3 +16,8 @@
 |:------------------|
 |Filename: [nvidia-dcgm.json](nvidia-dcgm.json)|
 |This dashboard has charts for viewing NVIDIA GPUs when monitored by Google's GPU enabled Ops Agent configured to use the DataCenter GPU Manager. It displays GPU utilization, GPU used memory, active GPU processes, Streaming Multiprocessor (SM) block utilization, SM occupancy, SM pipe utilization, PCIe traffic, and NVLink traffic.|
+
+|GCE & GKE GPU Utilization|
+|:------------------|
+|Filename: [gce-gke-gpu-utilization.json](gce-gke-gpu-utilization.json)|
+|This dashboard combines GCE and GKE GPU counts and utitilization.|
```

Comment:
utitilization -> utilization :)

Can you sync this description with the metadata.yaml one? Not super critical for them to be exactly the same but we're in the middle of a pipeline migration that parses the README to one that will parse the metadata.yaml.

Reply:
Thanks!

Reply:
Let me also sync the description.

Reply:
Done.